app/test_main.py

```diff
@@ -1,3 +1,42 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_human_age",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (29, 29, [3, 3]),
+        (100, 100, [21, 17]),
+        (-5, -5, [0, 0]),
+        (-100, -100, [0, 0]),
+    ],
+    ids=[
+        "Both zero",
```

Comment:
These descriptions must contain more information. Such as `Dog and Cat ages 14 must equal 0 human ages`